Task4.js

```diff
@@ -0,0 +1,73 @@
+//Objects and functions
+
+
+// -Создать объект car, добавить к нему свойство color со значением 'черный'
+// -Изменить свойство color объекта car на 'зеленый'
+// - В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
+let objCar = {
+    color: 'black'
+}
+objCar.color = 'green';
+console.log(objCar.color);
+objCar.power = () => {
+    console.log('The power of motor 1000');
+}
+objCar.power();
+
+
+// -На склад принимают груши и яблоки, напишите функцию, которая возвращает результат сложения количества принятых груш и яблок
+function fruits(apple, pear) {
+    if (apple < 0 || pear < 0) {
+        return 'check count!';
+    } else if (apple == 0 && pear == 0) {
+        return 'there are no fruits on the store';
+    } else if (apple > 0 || pear > 0) {
+        return `Count of apples and pears equal to ${apple + pear}`;
+    }
+
+}
+
+console.log(fruits(10, 10));
+
+
+// -В терминале оплаты сохранено ваше имя, напишите функцию для определения имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
+
+function terminal(name) {
```

Comment:
If you enter `IRINA` instead of `Irina` user would be not log in, but it is not a good idea to use case-sensitive.